ci/Jenkinsfile

```
@@ -70,16 +79,51 @@ pipeline {
   }
 
   stages {
-    stage('Download') {
+     stage('Download') {
       steps {
         script {
-          sh "mkdir '${env.WORKSPACE}/tmp/'"
-        }
-        fileOperations([
-          fileDownloadOperation(url:"${params.BUILD}", userName:"${CI_CREDS_USR}", password:"${CI_CREDS_PSW}", targetLocation:"./tmp", targetFileName:"StatusIm-Desktop.tar.gz", proxyHost:"", proxyPort:"")
-        ])
-        script {
-          sh "tar -zxvf '${env.WORKSPACE}/tmp/StatusIm-Desktop.tar.gz' -C '${env.WORKSPACE}/tmp'"
+          /* Creating directory for application build */
+          sh "mkdir -p '${env.WORKSPACE}/tmp/pkg/'"
+          def job_name = "${JOB_NAME}"
+          if(job_name.endsWith("nightly")){
+
+             /* Nightly regression */
+
+            if(params.BUILD_URL != ""){
+              echo "BUILD_URL value was ignored. Please use 'https://ci.infra.status.im/job/status-desktop/job/e2e/job/manual/' to test specific builds."
+            }
+            copyArtifacts(projectName: "status-desktop/systems/linux/x86_64/package/", filter: 'pkg/*-x86_64.tar.gz', selector: lastWithArtifacts(), target: "${env.WORKSPACE}/tmp")
```

Long lines are hard to follow.
```suggestion
            copyArtifacts(
              projectName: "status-desktop/systems/linux/x86_64/package/",
              filter: 'pkg/*-x86_64.tar.gz',
              selector: lastWithArtifacts(),
              target: "${env.WORKSPACE}/tmp"
            )
```
The same goes for any long call

---

Also, you're already in `WORKSPACE` folder, why do you need to specify it?

---

Why are you asking me about it? If you know how to make it in another way just write it here and say that it is better to use it like this. Let's save our time, "men"?